web/src/lib/components/forms/login-form.svelte

```diff
@@ -64,6 +65,11 @@
         },
       });
 
+      if (data.isAdmin && data.showOnboarding) {
+        dispatch('onboarding');
+        return;
+      }
+
       if (!data.isAdmin && data.shouldChangePassword) {
         dispatch('firstLogin');
         return;
```

Comment:
Should we refactor this `firstLogin` stuff to use the onboarding flag as well? (Or are we saving that for later?)

Reply:
I am thinking for later when we have an actual use case